include/radix/Window.hpp

```diff
@@ -22,58 +22,221 @@ class GWENInput;
 
 class Window : public Viewport, public InputSource {
 public:
+  /**
+   * @brief Window default constructor
+   */
   Window();
-  ~Window();
-  void create(const char*);
+
+  /**
+   * @brief ~Window default destructor
+   */
+  ~Window() override;
+
+  /**
+   * @brief create initialize SDL and initialize OpenGL
+   * 1. Initialize SDL system
+   * 2. Set OpenGL Version
+   * 3. Create SDL window
+   * 4. Create OpenGL Context
+   * 5. Initialize OpenGL
+   * 6. Set OpenGL Viewport
+   * 7. hide mouse
+   * 8. initlize other subsystem @see initGwen
+   * @param title window title
+   */
```

Comment:
@wow2006 As soon as there are code changes and the comment is not updated this comment will say something wrong. That is why the code is supposed to tell about the how for example **initSdlSystem(); setOpenGlVersion();** and the comments are for explaining the why.